src/main/java/com/google/api/codegen/discovery/transformer/js/JSSampleMethodToViewTransformer.java

```diff
@@ -98,6 +94,10 @@ private SampleView createSampleView(SampleTransformerContext context) {
       }
     }
 
+    if (methodInfo.isPageStreaming()) {
```

Comment:
Can we add a comment before this block in all 8 languages to note this issue? The order dependence is not immediately obvious.